lib/diagnostic.rb

```diff
@@ -35,7 +44,9 @@
 # "Somerville". Finally, assign the modified Person to `Response.dave` below.
 
 ## replace nil with your response, then continue your work on the next line
-dave = nil
+dave = Person.new("Dav", 32, "Ohio")
+  @location = "Somerville"
+
```

Comment:
Great job creating a new instance of Person!
to set the property value outside of the class we would use `dave.location = "Somerville"`